bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -8,6 +8,69 @@ components:
         - with-security
         - without-security
     bwc-test:
+      dependencies:
+        - job-scheduler
+        - alerting
+      test-configs:
+        - with-security
+        - without-security
+  - name: anomaly-detection
+    integ-test:
+      dependencies:
+        - job-scheduler
+        - alerting
```

Comment:
Why are all the plugins dependent on alerting and job-scheduler?

Reply:
As per the discussion https://github.com/opensearch-project/opensearch-build/pull/512#issuecomment-921975871.  I'll address it in the next PR.

Reply:
Sure. I am fine addressing this in a quick follow up PR. If so please create an issue so we have a track.

But always, as best practices: 
- Don't commit code we are not planning to use.
- Commit incorrect configuration which is un-usable.

Reply:
Yeah, that makes sense. Thanks for the new learning. I have create an issue to track this down: https://github.com/opensearch-project/opensearch-build/issues/521